eslint/babel-eslint-parser/src/convert/convertAST.cjs

```diff
@@ -105,6 +110,10 @@ function convertProgramNode(ast) {
   delete ast.program;
   delete ast.errors;
 
+  if (!ast.range) {
+    ast.range = [];
```

Comment:
When is this (and the other similar change at line 82) needed?

Reply:
I added this because the test that I've added was crashing without it.  
See my comment: https://github.com/babel/babel/issues/13331#issuecomment-844309140

Reply:
Removed in favour of https://github.com/babel/babel/pull/13338/commits/3769db3a70de3243b2bdcdd71f550a8074c03519